Bottom: ec68fac34157b640024587cdfa0021c5d8aae76c
Top:    cf13682c47e74cb9fa0e21578e771b0ebd1ca253
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 07:26:47 +0100

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 9e48d73..75b7eec 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
